backend/app/models/files.py

```diff
@@ -95,6 +109,7 @@ class FileDBViewList(View, FileBase):
     bytes: int = 0
     content_type: ContentType = ContentType()
     thumbnail_id: Optional[PydanticObjectId] = None
+    status: str = FileStatus.PRIVATE.name
 
```

Comment:
you already have it in the "FileBase", maybe no need to repeat here unless it's necessary?

Reply:
I removed this, it was not needed.